app/models/vote.rb

```diff
@@ -0,0 +1,12 @@
+class Vote < ApplicationRecord
+  validates :value, numericality: { only_integer: true,
+                                    odd: true,
```

Comment:
Why do you want this to be odd?

Reply:
Hi Brianna, the way we were figuring to implement the vote system was that a vote would only have a -1 or +1 value. If a user were to upvote something, and then upvote it again, then we would cancel their upvote by deleting the record entirely. So a vote of 0 is impossible and frees up database clutter when users cancel their votes. The only way I could figure how to do that was to make the range of a value be odd numbers between -1 and 1 so that 0 would be invalid. If there's a way to make a numericality validation be equal_to: 1 OR equal_to -1 then I would do it that way but I didn't see any way to do that online.